### DISCUSSION TITLE
State history / timeline data links should expose the duration

### DISCUSSION THREAD

**Discussion by TheSkorm** [grafana/grafana#67828] [Dashboards, Panels & Visualizations]
The state timeline feature allows for "Merge equal consecutive values". It would be useful to expose the duration of the merged values through to the data link configuration so that linked charts can show the represented data being linked from.

For example the below example the segment selected is 3 hours. It would be nice if this was exposed as a variable like `__value.duration`. Currently we have to use a fixed value and this confuses users who are expecting a much larger or smaller span.

<img width="467" alt="image" src="https://user-images.githubusercontent.com/234867/236193397-e4e38288-f2ba-4de6-8ebc-ae6f5e896ce6.png">
<img width="732" alt="image" src="https://user-images.githubusercontent.com/234867/236193508-562abfef-69e6-4b4b-a4b6-9adc14641516.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82671. Feel free to continue the discussion around this there. Thank you!
